.github/workflows/test.yml

```diff
@@ -5,35 +5,41 @@ name: test
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    name: Run tests (${{ matrix.os }}, pytorch version ${{ matrix.pytorch-version }}, python version ${{ matrix.python-version }})
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
         pytorch-version: ["master", "stable"]
+        python-version: [3.6, 3.7, 3.8]
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: "3.6"
-    - name: Install dependencies
-      run: |
-        if [[ ${{ matrix.pytorch-version }} = "master" ]]; then
-          pip install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html;
-        else
-          pip install torch==1.7+cpu -f https://download.pytorch.org/whl/torch_stable.html;
-        fi
-        pip install -e .[dev]
-    - name: Run unit tests
-      run: |
-        pytest --cov=kale
-    - name: Report coverage
-      run: |
-        coverage xml
-        bash <(curl -s https://codecov.io/bash)
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          if [[ ${{ matrix.pytorch-version }} = "master" ]]; then
+            pip install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html;
```

Comment:
We can remove this option. The stable version is what we use.

Reply:
Done.